packages/apps/composer-app/src/main.tsx

```diff
@@ -58,13 +61,24 @@ const main = async () => {
       // Core framework.
       ErrorPlugin(),
       GraphPlugin(),
+      MetadataPlugin(),
       ClientPlugin({ config, services, types }),
 
       // Core UX.
       LayoutPlugin(),
       NavTreePlugin(),
 
-      SpacePlugin(),
+      SpacePlugin({
+        onFirstRun: ({ personalSpaceFolder, dispatch }) => {
```

Comment:
`firstRun` comes from `ClientPlugin`, it's true when your identity and default space are being created, your "first run" of composer. `SpacePlugin` creates root folders when this variable is true and this callback allows the app to specify object it would like to place in those folders.